packages/core/core-flows/src/cart/workflows/list-shipping-options-for-cart-with-pricing.ts

```diff
@@ -61,11 +61,20 @@ export const listShippingOptionsForCartWithPricingWorkflow = createWorkflow(
         "sales_channel_id",
         "currency_code",
         "region_id",
-        "shipping_address.city",
-        "shipping_address.country_code",
-        "shipping_address.province",
-        "shipping_address.postal_code",
+        "shipping_address.*",
+        // ensure that the same subset of fileds needed to calculate shipping options is retrieved here and in calculateShippingOptionsPricesWorkflow when fetching cart
```

Comment:
thought(non-blocking): We could create a shared const for these and spread them in the existing fields arrays